lib/internal/Magento/Framework/App/Test/Unit/MaintenanceModeTest.php

```diff
@@ -44,9 +45,11 @@ protected function setup(): void
         $this->eventManager = $this->createMock(Manager::class);
 
         $objectManager = new ObjectManager($this);
+        $this->ipChecker = $objectManager->getObject(IpChecker::class);
```

Comment:
Please, declare this property explicitly.

Reply:
done